lib/csvlint/validate.rb

```diff
@@ -398,6 +398,16 @@ def fetch_error(error)
       rescue
         nil
       end
+
+      if message.match(/^Unquoted fields do not allow new line/i)
+        return :line_breaks
+      end
+
+      if message.match(/^New line must be/i)
+        return :inconsistent_line_breaks
+      end
```

Comment:
If I read the linter correctly `.match` does not return a boolean therefore it is creating a object for nothing.
To fix it, we should use `.match?`.

```suggestion
      if message.match?(/^Unquoted fields do not allow new line/i)
        return :line_breaks
      end

      if message.match?(/^New line must be/i)
        return :inconsistent_line_breaks
      end
```

Reply:
You're probably right, yes!